addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import fields, models, _
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdiProxyClientUser(models.Model):
+    _inherit = 'account_edi_proxy_client.user'
+
+    proxy_type = fields.Selection(
+        selection_add=[('peppol', 'PEPPOL')],
+    )
+
+    def _get_proxy_urls(self):
+        urls = super()._get_proxy_urls()
+        urls['peppol'] = {
+            'prod': '',
+            'test': 'https://l10n-pe-edi.test.odoo.com',
```

Comment:
:eyes:

Reply:
well, there is no prod url yet :laughing:

Reply:
prod is planned to be `peppol.api.odoo.com` and test `peppol.test.odoo.com` for now